sources/embed/efi/chan.c

```diff
@@ -17,7 +17,7 @@ IoResult embed_chan_write(IoChan channel, uint8_t const *data, size_t size)
 {
     assert_not_equal((int)channel, IO_CHAN_IN);
 
-    uint16_t *cstr = str_to_cstr_utf16_dos(str_n$(size, (char *)data), alloc_global());
+    uint16_t *cstr = utf16_str_to_cstr_dos(str_n$(size, (char *)data), alloc_global());
```

Comment:
dos ?

Reply:
Yes it's use in the UEFI bootloader and convert unix style line ending to dos like line ending

Reply:
okay, thank you for taking your time to answer my stupid questions